relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -210,14 +215,14 @@ mod tests {
 
         let session_metric = &metrics[0];
         assert_eq!(session_metric.timestamp, started());
-        assert_eq!(session_metric.name, "sentry.sessions.session");
+        assert_eq!(session_metric.name, "c:sessions/session@");
```

Comment:
In the proposal we had `c:sessions/session@none`. I'm fine with omitting the `none` when there is no unit, but should we omit the `@` as well?

Reply:
Right now, the none unit is still serialized with an empty string `""`, which should change. I left renaming the units for a followup PR, although that means that the MRI for those metrics will change shortly again. Would you prefer me to make the units change right away?

Reply:
We could also just map `""` to `"none"` in `new_mri` for now, but we should probably try to pin down the MRI right away.

Reply:
Given we would also have to rename `s` to `second` etc, I would prefer if we keep that change separately. Let's keep this PR open for another day, and I'll circle back here.